docs/dapps/integration/contract-integration.md

```diff
@@ -140,6 +140,10 @@ We recommend directing them to this page.
 
 :::
 
+## Combining Data Feeds
+
+The `data-feed-proxy-combinators` repository offers modular smart contracts for creating customized data feeds. These can be formed by transforming and combining existing feeds, which is useful when a specific data feed is not available on the Api3 Market. This allows dApps to integrate highly specific or composite data requirements directly into their smart contracts. See [here.](/dapps/integration/data-feed-proxy-combinators.md) for more details.
```

Comment:
```suggestion
which is useful when a specific data feed is not available on the Api3 Market
```

is confusing. I'd just remove this part of sentence.

Reply:
👍🏻 
Where should I put the link to `/dapps/integration/data-feed-proxy-combinators.md` ?